lib/src/components/NxTransferList/TransferListHalf.scss

```diff
@@ -85,23 +85,32 @@ $-visible-item-count: 10;
     #{$-visible-item-count - 1} * #{$-item-vertical-margin}
   );
 
-
   overflow-y: auto;
   padding: var(--nx-spacing-6x) $-control-box-side-spacing;
   margin-top: var(--nx-spacing-6x);
 }
 
 .nx-transfer-list__item {
   @include nx-container-helpers.container-horizontal;
-  @include nx-text-helpers.truncate-ellipsis;
 
   background-color: var(--nx-swatch-white);
-  border: var(--nx-border-default);
   border-radius: var(--nx-border-radius);
+  display: flex;
+  justify-content: space-between;
```

Comment:
What's this for?

Reply:
> What's this for?

Forgot to remove that: fixed: 8da7e29ba73263e6ce9f96e12d36ba9915305d0f